### ISSUE TITLE
Select project from map and show project details

### ISSUE THREAD

**Issue by TylerMatteo** [NYCPlanning/ae-cp-map#16]
## Acceptance Criteria
- [x] User can click or tap a project from the map and see the detail panel for that project ([Figma](https://www.figma.com/design/LYHHoPop9l0jpEivk5CFzJ/Capital-Projects-Portal?node-id=337-3601&t=i9OdRPOzwaKcOnqa-0))
- [x] Project detail panel is styled according to Figma designs for all screen sizes.
- [ ] When user selects a projects, map is panned so that the selected project is centered
- [x] Styling of project on the map updates to reflect selected state ([Figma](https://www.figma.com/design/LYHHoPop9l0jpEivk5CFzJ/Capital-Projects-Portal?node-id=670-5575&t=i9OdRPOzwaKcOnqa-0))

### Out of scope
- Showing [commitment-level data](https://www.figma.com/design/LYHHoPop9l0jpEivk5CFzJ/Capital-Projects-Portal?node-id=337-3772&t=i9OdRPOzwaKcOnqa-0) for the project. To be covered in #17 

## Blocked by
- #8 
- #18

**Comment by TangoYankee** [comment]
blocked by nycplanning/ae-zoning-api#248

**Comment by TylerMatteo** [comment]
> - [ ] When user selects a projects, map is panned so that the selected project is centered

Breaking this out to #49
